string rsrq5gnsa
string sinr5gnsa
string band5gnsa
string rsrp5g
string rsrq5g
string sinr5g
string band5g
